crates/swc_ecma_minifier/src/compress/optimize/mod.rs

```diff
@@ -280,6 +280,15 @@ impl Vars {
             ));
         }
 
+        if !self.lits.is_empty() {
+            n.visit_mut_with(&mut MultiReplacer::new(
```

Comment:
I'm not sure about potential performance cost about this, but I guess it's fine.